src/set_theory/cardinal_ordinal.lean

```diff
@@ -229,6 +230,47 @@ by rw [← aleph_zero, ← aleph_succ, ordinal.succ_zero]
 lemma countable_iff_lt_aleph_one {α : Type*} (s : set α) : countable s ↔ #s < aleph 1 :=
 by rw [← succ_omega, lt_succ, mk_set_le_omega]
 
+/-- Ordinals that are cardinals are unbounded. -/
+theorem ord_aleph'_unbounded : unbounded (<) {b : ordinal | b.card.ord = b} :=
```

Comment:
Is there a more appropriate name here? The type doesn't mention `aleph'`, nor the docstring.

Reply:
I'll go for `ord_card_unbounded`.